app/mailers/patient_mailer.rb

```diff
@@ -55,16 +60,21 @@ def assessment_sms_weblink(patient)
 
     add_success_history(patient)
     patient_name = "#{patient&.first_name&.first || ''}#{patient&.last_name&.first || ''}-#{patient&.calc_current_age || '0'}"
-    contents = "This is the Sara Alert system please complete the daily report for #{patient_name} at "
-    contents += new_patient_assessment_jurisdiction_report_url(patient.submission_token, patient.jurisdiction.unique_identifier).to_s
+    intro_contents = "This is the Sara Alert system please complete the report for #{patient_name} at the link provided"
+    url_contents = new_patient_assessment_jurisdiction_report_url(patient.submission_token, patient.jurisdiction.unique_identifier).to_s
```

Comment:
Only because I've seen that weird case where this failed in a template, want to do a nil (&.) check before .to_s